BAZProject/SearchMovie/SearchMovieViewController.swift

```diff
@@ -7,26 +7,28 @@
 
 import UIKit
 
-final class SearchMovieViewController: UICollectionViewController, Storyboarded {
+final class SearchMovieViewController: UICollectionViewController, Storyboarded{
     
     private let itemsPerRow: CGFloat = 2.0
     private let sectionInsets = UIEdgeInsets(top: 50.0, left: 20.0, bottom: 50.0, right: 20.0)
     private var movies: [Movie] = []
-    private let movieApi = MovieAPI()
+    private var likeMovieIndex: [Int] = []
+    private let movieApi = SearchAPI()
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        NotificationCenter.default.addObserver(self, selector: #selector(moviesCount(with:)), name: NSNotification.Name(rawValue: "contadorToVCSearch"), object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(moviesCount(_:)), name: .countMovieDetails, object: nil)
     }
     
     private func showMovieDetails(_ Element: Movie) {
```

Comment:
No usar Element con mayusculas. Para mi este método debería llamarse de la siguiente forma: 

`private func showDetails(for movie: Movie) {`

Reply:
También falta documentar.